### ISSUE TITLE
Missing manifest file

### ISSUE THREAD

**Issue by Tuned2aDeadChannel** [colin-szeto/Fusion-BOM-Scripts#2]
Missing manifest file.

**Comment by colin-szeto** [comment]
This is true. This repository was meant to supplement the workflow of Fusion360 not integrate as a plugin. Please feel free to take this code and iterate upon it to create the compatibility with Fusion360.

**Comment by Tuned2aDeadChannel** [comment]
Ok. Thank you for the response.
